### PR TITLE
Fix readme usage typo

### PR DESCRIPTION
**Pull request by kootoopas**

### CODE REVIEW

**Comment by timocov** [comment]
Hi @kootoopas,

Thanks for contributing to the project.

This codeblock contains a generated by running a command `dts-bundle-generator --help` command. Do you know if it possible to fix it there in the first place? So the next time we'll add or change a command we won't discard this change accidentally.

**Comment by kootoopas** [comment]
@timocov Hey,

Help option output is valid on v6.12.0 when run from command line. (`npx dts-bundle-generator --help`)

Extension exists solely on readme.

**Comment by timocov** [comment]
@kootoopas it looks like it depends on the system where you're running the command. On windows it has the extension:

```
$ npx dts-bundle-generator --help
Usage: dts-bundle-generator.js [options] <file(s)>
```

Probably we need to adjust [yargs usage command](https://github.com/timocov/dts-bundle-generator/blob/725d4f3b32c883658e0f56033b29b0cc54a6663d/src/bin/dts-bundle-generator.ts#L71) to address this. But I'm not too sure how it should be though.

**Comment by timocov** [comment]
Probably we can just remove that line from readme and leave yargs options as is (since they are accurate).

**Comment by kootoopas** [comment]
Given that the command can execute without the extension on windows, using the package name does produce constant output across OSs: [c9037b7](https://github.com/timocov/dts-bundle-generator/pull/216/commits/c9037b781f909fa1a83cc6e3bb836fd00dd4e086)

File: src/bin/dts-bundle-generator.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import * as yargs from 'yargs';
 
 import { loadConfigFile, BundlerConfig, ConfigEntryPoint } from '../config-file/load-config-file';
 
+import * as pkg from '../../package.json';
 import { generateDtsBundle } from '../bundle-generator';
```
Comment (timocov) [inline src/bin/dts-bundle-generator.ts:10]:
```suggestion
import { generateDtsBundle } from '../bundle-generator';
```

File: src/bin/dts-bundle-generator.ts [hunk 2]
```diff
@@ -68,7 +69,7 @@ function parseArgs(): ParsedArgs {
 			'short-option-groups': false,
 			/* eslint-enable @typescript-eslint/naming-convention */
 		})
-		.usage('Usage: $0 [options] <file(s)>')
+		.usage(`Usage: ${pkg.name} [options] <file(s)>`)
 		.demandCommand(0)
```
Comment (timocov) [inline src/bin/dts-bundle-generator.ts:73]:
```suggestion
		.usage('Usage: $0 [options] <file(s)>')
		.demandCommand(0)
```

**Review by timocov — APPROVED** [review]

**Comment by timocov** [comment]
Thanks @kootoopas! I think changes in the source code weren't necessary to I reverted them and left only changes in readme file.

**Pull request merged**